addons/zeus/functions/fnc_moduleSupplyDrop.sqf

```diff
@@ -0,0 +1,164 @@
+/*
+    @Authors
+        Christian 'chris5790' Klemm
+    @Project
+        Gruppe Adler Mod
+    @Arguments
+        ?
+    @Return Value
+        ?
+    @Example
+        ?
+*/
+#include "script_component.hpp"
+
+params ["_logic", "_units", "_activated"];
+
+if !(_activated && {local _logic}) exitWith {};
+
+private _box = attachedTo _logic;
+
+deleteVehicle _logic;
+
+if !(_box isKindOf "ReammoBox_F") exitWith {
+    [objNull, "Das Abwurfobjekt muss eine Ammobox sein"] call BIS_fnc_showCuratorFeedbackMessage;
+};
+
+if(_box getVariable [QGVAR(supplyDropInProgress), false]) exitWith {
+    [objNull, "Es ist gerade ein Abwurf mit dieser Box aktiv"] call BIS_fnc_showCuratorFeedbackMessage;
+};
+
+if(!createDialog QGVAR(moduleSupplyDrop)) exitWith {};
+
+GVAR(supplyBox) = _box;
+
+[] spawn {
+    disableSerialization;
+
+    private _dialog = findDisplay 3646235;
+    private _listbox = _dialog displayCtrl 1500;
+    private _confirmButton = _dialog displayCtrl 2400;
+    private _abortButton = _dialog displayCtrl 2401;
+
+    {
+        private _entry = _listbox lbAdd getText(_x >> "displayName");
+        _listbox lbSetData [_entry, configName _x];
+        _listbox lbSetPicture [_entry, getText(_x >> "picture")];
+    } forEach ("(configName _x) isKindOf 'Plane' && {getNumber(_x >> 'scope') == 2}" configClasses (configFile >> "CfgVehicles"));
+
+    lbSort _listbox;
+
+    _confirmButton ctrlAddEventHandler ["MouseButtonClick", {
+        _dialog = findDisplay 3646235;
+        _selection = lbCurSel (_dialog displayCtrl 1500);
+        if(_selection == -1) exitWith {};
+
+        GVAR(supplyPlaneType) = (_dialog displayCtrl 1500) lbData _selection;
+        closeDialog 0;
+
+        [
+            GVAR(supplyBox),
+            {
+                params ["_success", "_box", "_positionAsl"];
+                GVAR(supplyBox) = nil;
+                if(!_success) exitWith {
+                    GVAR(supplyPlaneType) = nil;
+                };
+                if(isNil {_box} || {isNull _box}) exitWith {};
+                if(_box getVariable [QGVAR(supplyDropInProgress), false]) exitWith {
+                    [objNull, "Es ist gerade ein Abwurf mit dieser Box aktiv"] call BIS_fnc_showCuratorFeedbackMessage;
+                };
+
+                _box setVariable [QGVAR(supplyDropInProgress), true, true];
+
+                [objNull, "Abwurf wird gestartet"] call BIS_fnc_showCuratorFeedbackMessage;
+
+                if(local _box) then {
```

Comment:
why so complicated, [bool] remoteExec ["allowDamage", _box]; should automatically execute on owner